tests/e2e/msg/proposals.go

```diff
@@ -0,0 +1,497 @@
+package msg
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/stretchr/testify/suite"
+
+	"cosmossdk.io/math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+
+	"github.com/cosmos/gaia/v23/tests/e2e/common"
+	"github.com/cosmos/gaia/v23/tests/e2e/data"
+)
+
+type Helper struct {
+	Suite suite.Suite
```

Comment:
I'm not sure why you're storing a suite here since it's seemingly not used.

Reply:
fixed